src/app/globals.css

```diff
@@ -2,24 +2,87 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer utilities {
+
+  /* Hide scrollbar for Chrome, Safari and Opera */
+  .no-scrollbar::-webkit-scrollbar {
+    display: none;
+  }
+
+  /* Hide scrollbar for IE, Edge and Firefox */
+  .no-scrollbar {
+    -ms-overflow-style: none;
+    /* IE and Edge */
+    scrollbar-width: none;
+    /* Firefox */
+  }
+}
+
+/*  fonts declaration */
+@font-face {
+  font-family: 'Basis Grotesque Mono Pro';
+  src: url('/fonts/BasisGrotesqueMonoPro-Regular.woff2') format('woff2');
+  font-weight: 400;
+  font-style: normal;
+}
+
+
 :root {
   --foreground-rgb: 0, 0, 0;
   --background-start-rgb: 214, 219, 220;
   --background-end-rgb: 255, 255, 255;
+  --basic-mono-font: basic-mono;
+  --header-height: 100px;
 }
 
-@media (prefers-color-scheme: dark) {
+/* @media (prefers-color-scheme: dark) {
   :root {
     --foreground-rgb: 255, 255, 255;
   }
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
 body {
   color: rgb(var(--foreground-rgb));
-}
+} */
+
 
 @layer utilities {
   .text-balance {
     text-wrap: balance;
   }
 }
+
+.select-dropdown {
+
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background-image: url('data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%22292.4%22%20height%3D%22292.4%22%3E%3Cpath%20fill%3D%22%23007CB2%22%20d%3D%22M287%2069.4a17.6%2017.6%200%200%200-13-5.4H18.4c-5%200-9.3%201.8-12.9%205.4A17.6%2017.6%200%200%200%200%2082.2c0%205%201.8%209.3%205.4%2012.9l128%20127.9c3.6%203.6%207.8%205.4%2012.8%205.4s9.2-1.8%2012.8-5.4L287%2095c3.5-3.5%205.4-7.8%205.4-12.8%200-5-1.9-9.2-5.5-12.8z%22%2F%3E%3C%2Fsvg%3E');
+    background-repeat: no-repeat, repeat;
+    background-position: right .7em top 50%, 0 0;
+    background-size: .65em auto, 100%;
+}
+.select-dropdown::-ms-expand {
+  display: none;
+}
+
+.selection-box {
+  position: relative; /* This makes the pseudo-element positioned relative to this element */
+  padding: 10px; /* Add padding inside the element */
+}
+
+.selection-box::before {
+  content: ""; /* This creates an empty pseudo-element */
+  position: absolute;
+  top: -10px; /* Creates space above the element */
+  left: -10px; /* Creates space to the left of the element */
+  right: -10px; /* Creates space to the right of the element */
+  bottom: -10px; /* Creates space below the element */
+  background-color: white; /* Transparent to avoid affecting visual design */
+  width: 100vw;
```

Comment:
The `width: 100vw;` in the psuedo element causes the content page to scroll horizontally.

switch to 100% width. The main concern is padding top and bottom to cut the content prematurely before getting to the selector

Reply:
Okay, but the 10px padding does that

Reply:
alright